sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -565,12 +565,10 @@ private synchronized Connection getOrCreateConnection() throws IOException {
             // remaining work after OperationTimeout has elapsed and closes afterwards.
             reactorProvider.getReactorDispatcher().getShutdownSignal()
                 .flatMap(signal -> {
```

Comment:
duplicated logging. will be logged in onConnectionShutdown and onConnectionError.